plugins/neo4j-graph-widget/components/WidgetNeo4jGraph.vue

```diff
@@ -96,8 +119,19 @@ import { default as Neo4jGraphImport } from '../components/Neo4jGraphImport.vue'
 import { default as polling } from '../core/mixin/polling'
```

Comment:
For info if the only export of the file is a default export, it can be written like this too:
```suggestion
import polling from '../core/mixin/polling'
```